_rest_api_collection/DE/Allgemeine_Infos_REST_API.md

```diff
@@ -19,7 +19,7 @@ Um den Webservice verwenden zu können, benötigst Du allerdings ein Autorisieru
 ## Schritte
 1. [Halte Dein Autorisierungstoken bereit](../../webui_collection/DE/Authentifizierungstoken).
 1. Erlange Zugang zur metasfresh Application Server REST API.
- >**Hinweis:** Die URL setzt sich in der Regel folgendermaßen zusammen:<br> https://`instanzname#`.metasfresh.com/app/swagger-ui.html#/
+ >**Hinweis:** Die URL setzt sich in der Regel folgendermaßen zusammen:<br> https://`instanzname#`.metasfresh.com/swagger-ui.html#/
```

Comment:
Hi @rene-dev, 
thanks a lot for taking the time!
But afaiu the documentation is correct as it is. 
In the standard docker-compose.yml based instance, 
`https://`instanzname#`.metasfresh.com/swagger-ui.html#/` points to the webui-api's swagger docs
whereas
`https://`instanzname#`.metasfresh.com/app/swagger-ui.html#/` points to the actual application server's swagger docs.

So in this case `app` is what this article is about..it's providing functional features like creating updating business partners.
On the other hand, the webui-api is working in close concert with the (javascript) fronted.

Reply:
ah, thanks! however, on app/swagger-ui.html/ I only get a blank page, no errors in the javascript console. both on the live demo, as well as in the docker container.